administrate.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.add_dependency "bourbon", "~> 5.0.0.beta.7"
   s.add_dependency "datetime_picker_rails", "~> 0.0.7"
   s.add_dependency "jquery-rails", "~> 4.0"
-  s.add_dependency "kaminari", "~> 0.16"
+  s.add_dependency "kaminari", "~> 1.0"
```

Comment:
What do you think about doing something like `>= 1.0` so we'll get 1.0 and above and no lower?